src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -120,8 +120,8 @@ def set_idrac_ip_address(self, ip_address):
 NODE_TEMPLATE_ATTRIBUTE_MODEL = 'model'
 NODE_TEMPLATE_ATTRIBUTE_SERVICE_TAG = 'service_tag'
 
-OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_PXE_IDRAC = 'pxe_drac'
-OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_PXE_IPMI = 'pxe_ipmitool'
+OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_PXE_IDRAC = 'idrac'
+OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_PXE_IPMI = 'ipmi'
```

Comment:
Please remove `PXE` from the constant names:

    OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_IDRAC = 'idrac'
    OSPD_NODE_TEMPLATE_VALUE_PM_TYPE_IPMI = 'ipmi'